request/util.go

```diff
@@ -0,0 +1,24 @@
+package request
+
+import (
+	"github.com/ONSdigital/go-ns/log"
+	"io"
+	"io/ioutil"
+	"net/http"
+)
+
+// DrainBody drains the body of the given of the given HTTP request.
+// any handler function that reads the request body should ensure that this
```

Comment:
"any" seems to assume that none will (or should) drain it themselves - strong words

Reply:
I have removed this line - I will leave it to the caller to decide if it's required or not 😄